### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Jeewal**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [ ] Resolved merge conflicts
- [ ] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

**Review by the-codingninja — COMMENTED** [review]
Can you provide a preview link

**Comment by Jeewal** [comment]
Preview link ? Of ? Sorry I do not understand what you are asking for.

**Review by the-codingninja — APPROVED** [review]

**Comment by abishekvashok** [comment]
What have u done in this pr, pls explain

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Pls add a static image to ./img/blogs/

**Comment by the-codingninja** [comment]
Yes it will be better if you add a static image

**Comment by Jeewal** [comment]
@abishekvashok  Explanation of the PR:
In this PR, I have the changes i made in the Index.html file. Basically, it is the change that will have my blog within the fossasia page with the photo i provided the source url of on the Blog portion of the fossasia page. (as the pr gets merged and the actual file in gci17.fossasia.org gets updated. ) 
https://jeewal.github.io/gci17.fossasia.org/
 Okay I will do that .

**Comment by abishekvashok** [comment]
Ok, pls do so :) good luck

**Review by anshumanv — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -565,6 +565,86 @@ <h5 class="uppercase bold mb16">Leave your phone</h5>
...
+		                            <h5 class="uppercase mb0">My dive into open source with FOSSASIA during Google Code-in</h5>
+		                            <span>Kuba Michalski</span>
+		                        </div>
+
+		                    </div>
+		                    </a>
+		                </div>
+
+		                <div class="col-md-3 col-sm-6">
+												<a href="https://saarthakchaturvedi05.wordpress.com/2017/12/06/theres-code-in-me/" target="_blank">
```
Comment (anshumanv) [inline index.html]: Can you please fix the indentation?

**Comment by the-codingninja** [comment]
Your path to the image seems wrong please check it

**Review by the-codingninja — CHANGES_REQUESTED** [review]
You have made two blog sections please merge it into one

**Comment by akshatnitd** [comment]
@Jeewal Please resolve the conflicts

**Review by the-codingninja — APPROVED** [review]

**Comment by the-codingninja** [comment]
Merging after checking the deployment link
Great Work!!
Cheers

**Pull request merged**